sourcegraph/cody
# Edit: Add codelens shortcuts

## Description

<img width="377" alt="image" src="https://github.com/sourcegraph/cody/assets/9516420/30f19019-dfb9-41b6-961a-eaffced03dd5">


This PR:
- Adds keyboard shortcuts to common Edit codelens actions (Accept, Retry, Undo, Cancel)
  - These shortcuts are only enabled when an Edit is in an active actionable state, and it is in a visible document. This will help reduce the risk of these shortcuts becoming an annoyance if a user doesn't use Edit much.
- Updates "Retry" text to show "Edit & Retry"


## Test plan

Single edit:

1. Create an edit
2. Try undo/retry/save shortcuts

Multiple edit (actions nearest edit):

1. Create multiple edits
2. Try undo/retry/save shortcuts and confirm that they only trigger against the nearest edit to the cursor position

Shortcut enablement: 

1. Create an edit
2. Leave the file
3. Try undo/retry/save shortcuts. Confirm nothing happens
4. Return to the file
5. Try undo/retry/save shortcuts. Confirm they work
